etl/steps/data/garden/owid/latest/key_indicators/utils.py

```diff
@@ -20,5 +21,5 @@ def add_regions(df: pd.DataFrame) -> pd.DataFrame:
     ]
     df = df.loc[-df.country.isin(regions)]
     for region in regions:
```

Comment:
Here we could use 
```
countries_in_region = geo.list_members_of_region(region=region, ds_regions=ds_regions, ds_income_groups=ds_income_groups)
df = geo.add_region_aggregates(df=df, region=region, countries_in_region=countries_in_region, population=population)
```
Where `ds_income_groups` should be the old hardcoded version of the `wb_income` dataset, and not the latest.